_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![image of me](https://user-images.githubusercontent.com/60928319/156002869-de5950e9-105b-40b1-9aac-201ae540609a.jpg)